minard/static/js/detector_state.js

```diff
@@ -393,41 +578,17 @@ function display_run_type(run_type,time_stamp) {
     if(det_state_desc.length > 0){
         appendToTitle('h3',det_state_desc);
     }
-    appendToTitle('p',time_stamp);
-};
-function display_mb(crateNum,cardNum,mb_data) {
-    var id ="#Crate"+crateNum+"MB"+cardNum
-    var thisCard = d3.select(id)
-    var size_info = {}
-    size_info['width'] = 200;
-    size_info['height'] = 100;
-    var width = size_info.width;
-    var height =size_info.height;
-    dv = thisCard.append("div");
-    dv.attr("class","LOOKHERE");
-    var svg = dv.append("svg")
-        .attr("width",width)
-        .attr("height",height)
-        .attr("viewBox","0 0 "+width.toString()+" "+height.toString())
-        .attr("class","crate_mask");
-    var step_size = 10;
-    var xpos_func = function(d,i) { return step_size+cardNum*step_size; }
-    var ypos_func = function(d,i) { return step_size+i*step_size; }
-    svg.selectAll("circle")
-        .data(mb_data['n100_triggers'])
-        .enter()
-        .append("circle")
-        .attr("cx",xpos_func)
-        .attr("cy",ypos_func)
-        .attr("r",5)
-        .attr("fill","red");
-};
-function display_crate_config(svg,crate_data) {
-    for (var i=0;i<16;i++) {
-        if(crate_data[i]) {
-            display_mb(1,i,crate_data[i]);
-        }
-    }
+
+    // The date object passed in from the DB has no timezone info,
+    // so JS assumes it's in GMT. So by printing it as a GMT time
+    // then editing the output timezone info in the string you
+    // end up at the right answer.
+    // Unfortunately the user has no way of seeing if the time is EST or EDT.
+    date = new Date(Date.parse(time_stamp))
```

Comment:
Since the time from the database will always be EST, you can use `moment.tz(time_stamp, "America/Toronto")`. See the docs here: http://momentjs.com/timezone/docs/#/using-timezones/parsing-in-zone/